ipatests/test_xmlrpc/test_cert_plugin.py

```diff
@@ -253,6 +253,43 @@ def test_00010_cleanup(self):
         res = api.Command['service_find'](self.service_princ)
         assert res['count'] == 0
 
+    def test_00011_email_are_valid(self):
+        """
+        Verify the different scenarios when checking if any email addr
+        from DN or SAN extension does not appear in ldap entry.
+        """
+
+        from ipaserver.plugins.cert import _emails_are_valid
+        from collections import namedtuple
+        NameAttr = namedtuple('NameAttr', 'value')
+
+        subject_addrs = [NameAttr(u'any@EmAiL.CoM')]
+        result = _emails_are_valid(subject_addrs, [], [u'any@email.com'])
```

Comment:
Done. I was waiting to we agree with the implementation and then change the tests.